README.md

```diff
@@ -34,120 +35,109 @@ This C# SDK is automatically generated by the [Swagger Codegen](https://github.c
 
 We recommend installing the package from [NuGet](https://docs.nuget.org/consume/installing-nuget):
 ```shell
-dotnet add package FingerprintPro.ServerSdk
+dotnet add package Fingerprint.ServerSdk
 ```
 
 <a name="getting-started"></a>
 ## Getting Started
 
 ```csharp
-// See https://aka.ms/new-console-template for more information
 // Example usage of our SDK
 
 // Import namespaces
-using FingerprintPro.ServerSdk.Api;
-using FingerprintPro.ServerSdk.Client;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Fingerprint.ServerSdk.Api;
+using Fingerprint.ServerSdk.Client;
+using Fingerprint.ServerSdk.Model;
 
-// Initialize configuration and add your api key
-var configuration = new Configuration(Environment.GetEnvironmentVariable("SECRET_API_KEY")!);
-
-var api = new FingerprintApi(
-    configuration
-);
-
-var requestId = Environment.GetEnvironmentVariable("REQUEST_ID")!;
-var visitorId = Environment.GetEnvironmentVariable("VISITOR_ID")!;
-
-var visits = api.GetVisits(visitorId);
-var events = api.GetEvent(requestId);
-var eventsFound = api.SearchEvents(20, paginationKey: "1740815825085", bot: "bad");
-
-Console.WriteLine(visits);
-Console.WriteLine(events);
-Console.WriteLine(eventsFound);
-
-var tag = new Tag
+namespace YourProject
 {
-    ["key"] = "value"
-};
-var body = new EventsUpdateRequest()
-{
-    Suspect = false,
-    Tag = tag,
-    LinkedId = "<linked_id>"
-};
-api.UpdateEvent(body, requestId);
-```
-
-You can also access the raw HTTP response by using the `WithHttpInfo` methods:
-
-```csharp
-// See https://aka.ms/new-console-template for more information
-// Example usage of our SDK
-
-// Import namespaces
-using FingerprintPro.ServerSdk.Api;
-using FingerprintPro.ServerSdk.Client;
-
-// Initialize configuration and add your api key
-var configuration = new Configuration(Environment.GetEnvironmentVariable("SECRET_API_KEY")!);
-
-var api = new FingerprintApi(
-    configuration
-);
-
-var requestId = Environment.GetEnvironmentVariable("REQUEST_ID")!;
-var visitorId = Environment.GetEnvironmentVariable("VISITOR_ID")!;
-
-var visits = api.GetVisitsWithHttpInfo(visitorId);
-
-// HttpResponseMessage
-Console.WriteLine(visits.Response);
+    public class Program
```

Comment:
Can we have a complete scenario here and not just the API configuration for the DI?